src/com/massivecraft/massivetickets/cmd/CmdTicketsTeleport.java

```diff
@@ -30,7 +30,7 @@
 	public CmdTicketsTeleport()
 	{
 		// Parameters
-		this.addParameter(TypeMPlayer.getAny(), "player");
+		this.addParameter(TypeMPlayer.getOnline(), "player");
```

Comment:
Why is this a good idea? MassiveSharding supports teleporting to the location of the offline player, does this not work ingame right now?

Reply:
Wasn't aware of this functionality with MassiveSharding; had never considered could be a possibility after only local testing solely with MassiveTickets. Will revise.